op-plasma/cmd/daserver/entrypoint.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/ethdb/leveldb"
+	levelopt "github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/urfave/cli/v2"
+
+	plasma "github.com/ethereum-optimism/optimism/op-plasma"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum-optimism/optimism/op-service/opio"
+)
+
+func Main(cliCtx *cli.Context) error {
+	if err := CheckRequired(cliCtx); err != nil {
+		return err
+	}
+
+	cfg := ReadCLIConfig(cliCtx)
+	if err := cfg.Check(); err != nil {
+		return err
+	}
+
+	logCfg := oplog.ReadCLIConfig(cliCtx)
+
+	l := oplog.NewLogger(oplog.AppOut(cliCtx), logCfg)
+	oplog.SetGlobalLogHandler(l.Handler())
+
+	l.Info("Initializing Plasma DA server...")
+
+	var store plasma.KVStore
+
+	if cfg.LevelDBEnabled() {
```

Comment:
Another preference is to use raw files instead of levelDB. We can use the commitment as the file name.